generic_config_updater/gu_common.py

```diff
@@ -65,16 +66,17 @@ def get_config_db_as_json(self):
         return config_db_json
 
     def _get_config_db_as_text(self):
-        if self.namespace is not None and self.namespace != multi_asic.DEFAULT_NAMESPACE:
-            cmd = ['sonic-cfggen', '-d', '--print-data', '-n', self.namespace]
+        if self.scope is not None and self.scope != multi_asic.DEFAULT_NAMESPACE:
+            cmd = ['sonic-cfggen', '-d', '--print-data', '-n', self.scope]
```

Comment:
I notice before commit of "fix relplacer", it is wrongly using `self.namespace` but didn't report any error. Can we add a unit test to catch this?

Reply:
Fair asking, added.